web/src/components/LocaleSwitcher.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <v-select
+    v-model='$i18n.locale'
+    prepend-icon='fa-globe'
```

Comment:
Let's use material icons for consistency. I recommend the ['language' icon](https://material.io/resources/icons/?icon=language&style=baseline).